packages/graphback-codegen-resolvers/src/templates/createResolvers.ts

```diff
@@ -1,8 +1,29 @@
-import { getFieldName, getSubscriptionName, GraphbackCRUDGeneratorConfig, GraphbackOperationType, ModelDefinition } from '@graphback/core';
+import { getFieldName, getSubscriptionName, GraphbackCRUDGeneratorConfig, GraphbackOperationType, ModelDefinition, RelationshipMetadata, getPrimaryKey } from '@graphback/core';
 import { GraphQLObjectType } from 'graphql';
 import { GeneratorResolversFormat } from '../GeneratorResolversFormat';
-import { createTemplate, deletedSubscriptionTemplate, deleteTemplate, findAllTemplate, findTemplate, newSubscriptionTemplate, updatedSubscriptionTemplate, updateTemplate } from './resolverTemplates';
- 
+import { createTemplate, deletedSubscriptionTemplate, deleteTemplate, findAllTemplate, findTemplate, newSubscriptionTemplate, oneToOneTemplate, updatedSubscriptionTemplate, updateTemplate, oneToManyTemplate } from './resolverTemplates';
+
+export function createRelationshipResolvers(relationships: RelationshipMetadata[]) {
+    if (!relationships.length) { return undefined; }
+
+    const resolvers = {};
+    for (const relationship of relationships) {
+        let resolverOutput: string;
+        const relationTypeName = relationship.relationType.name;
+        const relationIdField = getPrimaryKey(relationship.relationType);
```

Comment:
Method name here is confusing. Are we simply repurposing PrimaryKey method to get foreignKey?

Reply:
We are getting the primary key field for the relation type.

Reply:
Which translateds to getForeginKey name